cmd/shortener/main.go

```diff
@@ -116,22 +117,35 @@ func main() {
 	if conf.Cfg.DataBaseDNS == "" {
 		conf.Cfg.DataBaseDNS = *d
 	}
+	sh := ShortenerHandler{}
 	if conf.Cfg.DataBaseDNS != "" {
 		ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
 		defer cancel()
-		db, err := storage.NewDBStorage(conf.Cfg.DataBaseDNS, ctx)
+		storage, err := database.NewDBStorage(conf.Cfg.DataBaseDNS, ctx)
 		if err != nil {
 			log.Println(err.Error())
```

Comment:
```suggestion
			log.Fatal(err)
```
Если прилетела ошибка, то storage будет nil. И любые попытки обратиться к его функциям, будут приводить к панике. Так что все равно придется останавливать и перезапускать сервер.